File: Beginner/Java/1008/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
    	Scanner scanner = new Scanner(System.in);
        int number = scanner.nextInt();
        int hours = scanner.nextInt();
        float hourValue = scanner.nextFloat();
        
        System.out.printf("NUMBER = %d%nSALARY = U$ %.2f%n", number, (hourValue*hours));
    }

}

File: Beginner/C/1008/main.c
#include <stdio.h>
 
int main() {
    int number, hours;
    float hourValue;
    scanf("%d", &number);
    scanf("%d", &hours);
    scanf("%f", &hourValue);

    printf("NUMBER = %d\nSALARY = U$ %.2f\n", number, (hourValue*hours));
    return 0;
}
